aries-site/src/examples/components/index.js

```diff
@@ -36,6 +37,7 @@ export * from './tabs';
 export * from './table';
 export * from './textarea';
 export * from './textinput';
+export * from './toast-notifications';
```

Comment:
should we move these to the templates folder since that is where "Toast Notifications" page is?